### DISCUSSION TITLE
Holes in the brain model of stroke patients

### DISCUSSION THREAD

**Discussion by enemonsieur** [simnibs/simnibs#476] [9. Results and analysis] [answered]
I've been struggling with modelling some brain tissues. It seems like it only creating a mesh of the surface, and the other part of the brain don't seem to appear at all. It's as if some parts like the grey matter, etc... weren't consider. 

I used charm, changed the setting.nii from "atlas" to "mni". And ran the run_simnibs(S). I also added a stroke lesion using the command of simnibs.

Did anyone encouter that sort of issue? Or should I add more details?

Best!
![image](https://github.com/user-attachments/assets/912fa7ea-ab14-469c-ab79-968001d0df18)

**Comment by TorgeW** [comment, marked as answer]
You should first check the segmentation of the head. For this take a look at label_prep/tissue_labeling_upsampled.nii.gz and check that everything looks as you would expect.

For the visualization please check https://simnibs.github.io/simnibs/build/html/tutorial/visualization.html
You can look at the tetrahedra instead of the surfaces by going to View-> and only enable tissue 2.  Then go to Options-> Mesh-> Enable 3d element faces.

**Reply by enemonsieur** [reply]
Thank you. Indeed I didn't add the tetra and it didn't register. By the way, the holes in the brain were caused because of the big lesion in the brain. Probably because I didn't model the lesion correctly.
